cgcnn2/cli/cgcnn_ft.py

```
@@ -319,6 +319,9 @@ def main():
     normalizer = Normalizer(torch.zeros(3))
     normalizer.load_state_dict(checkpoint["normalizer"])
 
+    metrics = []
+    if mse is not None:
```

Ruff (0.11.9)

Undefined name `mse`

(F821)